src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_copy_scenarios.py

```diff
@@ -106,3 +106,28 @@ def test_storage_blob_copy_same_account_sas(self, resource_group, storage_accoun
             actual_content = f.read()
 
         self.assertEqual(expect_content, actual_content)
+
+        # test sas-token input starting with '?'
+        if not sas.startswith('?'):
+            sas = '?' + sas
+
+        target_container = self.create_container(account_info)
+        self.storage_cmd('storage blob copy start -b dst -c {} --source-blob src --sas-token {} --source-container {}',
+                         account_info, target_container, sas, source_container)
```

Comment:
Does this actually test the sas?  The `storage_cmd` append storage account and key to the command, won't it disrupt the intention of this test?

Reply:
You're right, the validator for the "regular" destination storage account arguments can/do parse the '?' from the sas token argument. 
When the validator for the source sas copies the token from the destination, it receives the parsed token without the query parameter.

I'll change this test to use the `--source-sas` argument.